### DISCUSSION TITLE
Remove the 50 shades of grey and add more contrast

### DISCUSSION THREAD

**Discussion by Siedlerchr** [community/community#70255] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

Sorry: Angry post incoming.

Stop playing around with useless design decisions that hurt accessibility:

Do you even use your website? 
<img width="1678" alt="grafik" src="https://github.com/community/community/assets/320228/6be1b0d9-35bf-438d-b64a-c43d67fa8c1c">

It's all 50 shades of grey and hardly visible. 
Please revert your changes. Not everyone uses dark themes

And stop moving the search to the right. You tried this years ago already and it was bad. And now you do it again?
Sorry but this sucks!

**Comment by ghostinhershell** [comment]
Hi @Siedlerchr, thanks so much for sharing your feedback here. I agree with your point here that not everyone uses dark themes and that it isn't a one size fits all approach for users who need or even prefer higher contrast web elements. I know we have some light themes that include more contrast that may do the trick for you. For example, I personally like to use the "Light - High Contrast" theme when working during the day:
![image](https://github.com/community/community/assets/14021469/d156459e-00fd-4e1e-8432-1116d60c7b7b)

----
I can still log the feedback that you've shared here but can you try some of our other light colored themes and let me know if that helps?

**Reply by ghostinhershell** [reply]
Hi @Siedlerchr, just wanted to follow-up here. Have you gotten a chance to try out using our other themes to see if that improves visibility for you?

**Comment by cehfisher** [comment]
Hi @Siedlerchr - I wanted to follow up on this stale issue. Have the recent theme changes resolved this issue for you? If yes, could you please close the issue? If no, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)

**Reply by Siedlerchr** [reply, marked as answer]
Yes, it looks a bit better now
